### ISSUE TITLE
Don't use gunicorn

### ISSUE THREAD

**Issue by andreymal** [startmatter/python-frameworks-benchmark#1]
https://docs.aiohttp.org/en/stable/deployment.html

>nothing is for free: running aiohttp application under gunicorn is **slightly slower**.

Sanic and aiohttp have nice asynchronous built-in servers that are production-ready. You should use them instead of gunicorn

**Comment by maximzah** [comment]
@andreymal Thank you for your interest, we initially aimed at testing frameworks and methods of starting. It is clear that it is possible to select the optimal launch parameters for each framework for a long time - but our task was to test all frameworks in **equal conditions** - therefore we chose gunicorn.

But your comment "woke up" our interest and we tested aiohttp without gunicorn in the [aiohttp_without_gunicorn](https://github.com/startmatter/python-frameworks-benchmark/tree/aiohttp_without_gunicorn) branch and this is the result (as you can see, the difference is not significant)

![image 6](https://user-images.githubusercontent.com/17846448/52843588-a2624680-3113-11e9-8c81-fa243082376a.png)

**Comment by andreymal** [comment]
@maximzah 

>we tested aiohttp without gunicorn in the [aiohttp_without_gunicorn](https://github.com/startmatter/python-frameworks-benchmark/tree/aiohttp_without_gunicorn)

Why I still see gunicorn in the Dockerfile? https://github.com/startmatter/python-frameworks-benchmark/blob/aiohttp_without_gunicorn/aiohttp/Dockerfile

>but our task was to test all frameworks in **equal conditions**

![](https://i.imgur.com/eKr6UQN.jpg)

**Comment by andreymal** [comment]
Okay, I ran aiohttp without gunicorn in Ubuntu 18.04 (Intel Core i3-3220) and the difference is really not significant (2204 req/sec vs. 2171 with workers=1)

But I also ran sanic without gunicorn and it's 1.5x faster! (2832 vs. 1920 with workers=1)

(I need to use a load balancer like nginx for workers=2 but I'm too lazy today)

**Comment by VladimirRubin** [comment]
@andreymal Thank you very much for researching performance of this two frameworks with another configs 👍 

I think it would be great to share this with the community and if you have a desire to do this - you can find the ideal startup configurations for each framework in the repository and create PR.

Welcome to Open Source 🤝

**Comment by stalkerg** [comment]
Yes, for the normal async framework you shouldn't use any proxy servers. (it needs only for thread/process like fat frameworks)
